Services/DataBaseService/DbModels/DbUser.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace DataBaseService.DbModels
+{
+    public class DbUser
+    {
+        public Guid Id { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+    }
+
+    public class UserConfiguration : IEntityTypeConfiguration<DbUser>
+    {
+        public void Configure(EntityTypeBuilder<DbUser> builder)
```

Comment:
What will happen if some of them aren't correct?

Reply:
If something of them aren`t correct, database throws exception. Add try catch when call execute method in controller

Reply:
UPD: remove to Create method in UserCredentialRepository